### PR TITLE
License and source headers update

### PR DESCRIPTION
**Pull request by zippy1978**
Work done:
- changed license from LGPL v3 to GPL v3
- updated org name to `green-code-initiative`
- updated all source headers

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
     <name>ecoCode iOS plugin</name>
     <description>Help the earth, adopt this green plugin for your applications</description>
-    <inceptionYear>2022</inceptionYear>
+    <inceptionYear>2023</inceptionYear>
     <organization>
-        <name>Green code Initiative</name>
+        <name>green-code-initiative</name>
         <url>https://collectif.greenit.fr</url>
```
Comment (dedece35) [inline pom.xml:15]: change also organization URL please

**Review by dedece35 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (zippy1978) [reply pom.xml:15]: With this one :  https://github.com/green-code-initiative ?

Comment (zippy1978) [reply pom.xml:15]: No it's fine !
I do it !

**Review by dedece35 — APPROVED** [review]

**Comment by utarwyn** [comment]
Any news on this? Maybe we can merge and create a release?

**Comment by dedece35** [comment]
> Any news on this? Maybe we can merge and create a release?

Hello @utarwyn and @zippy1978 ... ok for me.
ok for you @zippy1978 to merge ?

**Comment by zippy1978** [comment]
Yes,
I merge it

**Comment by dedece35** [comment]
@zippy1978 if github action is ok, are you ok that I create a release ?

**Comment by zippy1978** [comment]
Yes, go ahead @dedece35 !

**Comment by dedece35** [comment]
in progress ... :p

**Comment by dedece35** [comment]
hi @utarwyn @zippy1978 @olegoaer @jhertout @glalloue @utarwyn 
after some updates / corrections with release process int this repository, the first release is OK with 1.1.0 version of ios plugin :p

**Pull request merged**